ui/components/app/hold-to-reveal-button/hold-to-reveal-button.js

```diff
@@ -152,6 +152,7 @@ export default function HoldToRevealButton({ buttonText, onLongPressed }) {
           </svg>
         </div>
         <div
+          aria-label="hold to reveal circle unlocked"
```

Comment:
AriaLabels should be localized; please add a new entry to `app/_locales/en/messages.json` for this string.

Reply:
sure